src/Rings/groebner.jl

```diff
@@ -1048,6 +1080,13 @@ function normal_form(A::Vector{T}, J::MPolyIdeal; ordering::MonomialOrdering=def
   return res
 end
 
+function is_normal_form_f4_applicable(I::MPolyIdeal, ordering::MonomialOrdering)
+    return (ordering == degrevlex(base_ring(I)) && !is_graded(base_ring(I))
+            && ((typeof(coefficient_ring(I)) == FqField
```

Comment:
```suggestion
            && ((coefficient_ring(I) isa FqField
```
Trivial change, mainly so I can make sure GitHub *really* restarts the tests.